### PR TITLE
max size limit: 300 + force plot

### LINKED ISSUE #10: Warning for setting too high of a size.
Some users set the size too high and then their browser starts to freeze.

@biologyscience, Can you add a warning when the size exceeds 200? Maybe ask for confirmation.

I don't know if its possible in java script.

### PR DESCRIPTION
**Pull request by biologyscience**
closes #10 

graph will not be plotted + warning

### CODE REVIEW

**Comment by tusharhero** [comment]
Can the user plot it anyway. If they wanted @biologyscience ?

**Comment by biologyscience** [comment]
> Can the user plot it anyway. If they wanted @biologyscience ?

no

but do you want like a force to plot kinda thing ?

**Comment by tusharhero** [comment]
> > Can the user plot it anyway. If they wanted @biologyscience ?
> 
> no

Let them plot if they still want even after the warning.

> but do you want like a force to plot kinda thing ?

Yup, the power to overwrite.

**Comment by biologyscience** [comment]
but when you force plot after size 300, the graph doesn't plot
reason: brython
proof: image

**Comment by tusharhero** [comment]
> but when you force plot after size 300, the graph doesn't plot reason: brython proof: image

what? It works without your patches tho? @biologyscience

**Comment by tusharhero** [comment]
[proof here](https://tusharhero.github.io/asciificationpy/?display=YXhlcyh4LHkpL3wzMDEvfDE=)

File: index.py [hunk 1]
```diff
@@ -2,38 +2,66 @@
...
-    document["output"].value = ""
-    document["error"].html = "$ I ate the graph"
 
 def startup():
-    eqt = document["-eqt"].value
-    size = document["-size"].value
-    scaling = document["-scale"].value
 
-    if eqt == 'default' or size == 'default' or scaling == 'default':
+    if document["-eqt"] == 'default' or document["-size"] == 'default' or document["-scale"] == 'default':
```
Comment (tusharhero) [inline index.py:52]: I think it should be `document["-foo"].value`.

File: index.py [hunk 2]
```diff
@@ -2,38 +2,66 @@
...
+        eqt = checkData[0]
+        size = checkData[1]
+        scale = checkData[2]
+
+        graph = asc.getstring(ascp.plot(eqt, img=asc.createblankimg(size, size), s=scale))
         document["error"].html = "$ checking"
         document["box"].width = size
         document["output"].value = graph
         document["error"].html = "$ done!"
-    except Exception as e:
```
Comment (tusharhero) [inline index.py:15]: why did you get rid of this? It is used to display any python errors directly in the webpage.

**Comment by biologyscience** [comment]
alr whatever, make the changes u want. imma head out

**Comment by tusharhero** [comment]
So you are not going to add this feature? @biologyscience

**Comment by tusharhero** [comment]
@biologyscience I tested your pr in local env. It gives the warning after the plot has already been plotted.

**Pull request merged**